compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_libcdep.cpp

```diff
@@ -108,6 +108,7 @@ bool Symbolizer::SymbolizeData(uptr addr, DataInfo *info) {
                                          &arch))
     return false;
   info->Clear();
+  info->start = addr;
```

Comment:
Why?
If there is a reason please extract into another PR, as this change likely affect other sanitizers.
Would be possible to have a test?

Reply:
> Why? If there is a reason please extract into another PR, as this change likely affect other sanitizers. Would be possible to have a test?

This change is not necessary for this PR and I will remove it. Thanks.